StreamingKit/StreamingKit/STKAudioPlayer.m

```diff
@@ -2133,6 +2169,22 @@ -(void) createAudioConverter:(AudioStreamBasicDescription*)asbd
     }
 }
 
+-(void) createPlaybackRateUnit
+{
+    OSStatus status;
+    
+    CHECK_STATUS_AND_RETURN(AUGraphAddNode(audioGraph, &playbackRateUnitDescription, &playbackRateNode));
+    CHECK_STATUS_AND_RETURN(AUGraphNodeInfo(audioGraph, playbackRateNode, &playbackRateUnitDescription, &playbackRateUnit));
+    
+    //maxframes changed here
+    CHECK_STATUS_AND_RETURN(AudioUnitSetProperty(playbackRateUnit, kAudioUnitProperty_MaximumFramesPerSlice, kAudioUnitScope_Global, 0, &maxFramesPerSlice, sizeof(maxFramesPerSlice)));
+#if TARGET_OS_IPHONE
+    CHECK_STATUS_AND_RETURN(AudioUnitSetParameter(playbackRateUnit, kNewTimePitchParam_Rate, kAudioUnitScope_Global, 0, 1, 0));
+#endif
+    
+  CHECK_STATUS_AND_RETURN(AudioUnitSetProperty(playbackRateUnit, kAudioUnitProperty_StreamFormat, kAudioUnitScope_Input, kOutputBus, &canonicalAudioStreamBasicDescription, sizeof(canonicalAudioStreamBasicDescription)));
+}
```

Comment:
This exact method introduces a crash in some rare cases (but reproducible):

<img width="934" alt="image" src="https://user-images.githubusercontent.com/5748627/70802119-4317a600-1db9-11ea-9c31-f27e18505e17.png">

Do you have any idea of why would it crash there?

Reply:
@pcbeard, Do you have any idea?